src/java/picard/util/BedToIntervalList.java

```diff
@@ -29,7 +29,19 @@
  * @author nhomer
  */
 @CommandLineProgramProperties(
-        usage = "Converts a BED file to an Picard Interval List.",
+        usage = "Converts a BED file to an Picard Interval List.  Bed file is a flexible format used to define data" +
```

Comment:
> Bed file is a flexible format

Technically, a file is not a format... Also, if we want to explain what BED is, could be good to spell it out.